ee/clickhouse/migrations/0029_person_version_column.py

```diff
@@ -0,0 +1,12 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.person import KAFKA_PERSONS_TABLE_SQL, PERSONS_TABLE_MV_SQL
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+operations = [
```

Comment:
I think this may have led to dropped messages from Kafka as it's expecting version to always be there which it would not be pre-deploy.

Please confirm if this is a problem - if it is, we should revert

Reply:
Verified this myself - no, Kafka assumes version == 0 through the pipe. This is still an issue as we're relying on undefined behavior here.

Note that the question on when/how we can switch to collapsing by version is still out there. Currently deletion would result in the wrong version in the table, and most of the rows in the db will have incorrect versioning. Will create follow-up issues.